src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -397,9 +399,9 @@ class ResourceService(private val resourceTypes: Map[ResourceTypeName, ResourceT
     for {
       accessPolicies <- accessPolicyDAO.listAccessPolicies(resourceId)
       members <- accessPolicies.toList.parTraverse(accessPolicy => accessPolicyDAO.listFlattenedPolicyMembers(accessPolicy.id))
-      workbenchUsers <- directoryDAO.loadUsers(members.toSet.flatten)
+      workbenchUsers = members.flatten.toSet
```

Comment:
just fyi, you can do 
```
      members <- accessPolicies.toList.parFlatTraverse(accessPolicy => accessPolicyDAO.listFlattenedPolicyMembers(accessPolicy.id).map(_.toList))
```
This will save you `flatten`...but with `parFlatTraverse`, you'll need that extra `.map(_.toList))`